.github/workflows/ci.yml

```diff
@@ -113,3 +121,42 @@ jobs:
         run: |
```

Comment:
add

```yml
      - name: Checkout code
        uses: actions/checkout@v3
      - name: Set ENV for codeclimate (pull_request)
        run: |
          git fetch --no-tags --prune --depth=1 origin refs/heads/$GITHUB_HEAD_REF:refs/remotes/origin/$GITHUB_HEAD_REF
          echo "GIT_BRANCH=$GITHUB_HEAD_REF" >> $GITHUB_ENV
          echo "GIT_COMMIT_SHA=$(git rev-parse origin/$GITHUB_HEAD_REF)" >> $GITHUB_ENV
        if: github.event_name == 'pull_request'
```
as the first step of the `coverage` job and second step of the `tests` job